content/articles/2018-12-14-tidyverse-and-r-lib-year-in-review.Rmarkdown

```diff
@@ -0,0 +1,45 @@
+---
+title: "tidyverse and r-lib: a year in review"
+author: Mara Averick
+date: '2018-12-14'
+slug: tidyverse-and-r-lib-year-in-review
+description: > 
+  A look back on 2018 in the tidyverse and beyond.
+categories:
+  - other
+photo:
+  url: https://unsplash.com/photos/SshYpuf607g
+  author: Aperture Vintage
+---
+
+*Hadley Wickham, Mara Averick, Jenny Bryan, Gábor Csárdi, Romain François, Alex Hayes, Lionel Henry, Jim Hester, Max Kuhn, Thomas Lin Pedersen, Davis Vaughan*
```

Comment:
I'd replace this with some sort of lead-in: It's been a banner year for the tidyverse and r-lib teams and as we close out the year we wanted to take a look back at some of our achievements.